### PR TITLE
[type:feat] add web socket plugin and add proxy ping configuration

### PR DESCRIPTION
**Pull request by wenlongbrother**
#5142 
1. add proxy ping configuration field.
2. add the filed to WebsocketClientSpec in org.apache.shenyu.springboot.plugin.websocket.WebSocketPluginConfiguration

Make sure that:

- [X] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [X] You submit test cases (unit or integration tests) that back your changes.
- [X] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by moremind** [comment]
pls add the config in `application.yml`

File: shenyu-common/src/main/java/org/apache/shenyu/common/config/ShenyuConfig.java [hunk 1]
```diff
@@ -1549,7 +1549,12 @@ public static class WebsocketConfig {
          * max frame pay load size mb.
          */
         private Integer maxFramePayloadSize = 10;
-    
+
+        /**
+         * whether enable ping.
+         */
+        private Boolean enablePing = false;
```
Comment (moremind) [inline shenyu-common/src/main/java/org/apache/shenyu/common/config/ShenyuConfig.java:1556]: may `enableProxyPing` adapter the scene?

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-common/src/main/java/org/apache/shenyu/common/config/ShenyuConfig.java (hunk 1 above)
Comment (wenlongbrother) [reply shenyu-common/src/main/java/org/apache/shenyu/common/config/ShenyuConfig.java:1556]: ok, will do thank you.

**Review by moremind — APPROVED** [review]

**Comment by moremind** [comment]
pls add my wechat: yijia1949

**Comment by wenlongbrother** [comment]
Hi @moremind can you help check k8s check? I see it failed but I am not sure what happend

**Pull request merged**